common/api/core-common.api.md

```diff
@@ -227,8 +233,11 @@ export class AnalysisStyleThematic {
 
 // @public
 export interface AnalysisStyleThematicProps {
+    // (undocumented)

```

Comment:
Many properties are now incorrectly being marked "undocumented" because their documentation consists of "@see [[SomeOtherProperty]]". [Example](https://github.com/iTwin/itwinjs-core/blob/f4cdf70998c472486ee3ec07d1d2606f811fdba2/core/common/src/AnalysisStyle.ts#L71)
We do this all over the place for Props interfaces that have a corresponding class to avoid potentially error-prone duplication of documentation.

Reply:
For some reason it seems api-extractor isn't recognizing the `@see` comment syntax which it should since api-extractor uses the TSDoc grammar 🤔. The `// (undocumented)` addition does go away if I change `@see` to just `See`.

Is there a reason the property is wrapped around `[[]]`? It seems that the wrapping is preventing from the (cmd/ctrl)+click feature as mentioned [here](https://www.typescriptlang.org/play?ts=4.1.0-beta#example/see-in-jsdoc). This makes me wonder if we can change `@see` into `See` and still prevent duplication of documentation?

Reply:
`[[Something]]` is a shortcut for `[Something]($name-of-this-package)`.
I believe these two syntaxes belong to BeMetalSmith, not TSDoc -- so perhaps new version of api-extractor is ignoring them because they do not conform to TSDoc grammar.
Do you want to experiment by replacing one of these occurrences with something like `@see {@link https://tools.ietf.org/html/rfc1738|RFC 1738}` and see if it still gets marked as undocumented?
@williamkbentley

Reply:
Yes. Double square brackets was a syntax that TypeDoc used to support, so BeMetalsmith supports it as well. But it was insufficient for cross-package linking. Which is why we have `$package-name` as well. 

Seems to me that TSDoc doesn't consider a single `@see` tag to be documented?

Reply:
That's what the proposed experiment is supposed to tell us.

Reply:
I tried this: 
![image](https://user-images.githubusercontent.com/98850418/179265368-7a98b75b-de28-408b-832c-2a0e485da09a.png)
and this:
![image](https://user-images.githubusercontent.com/98850418/179265824-4335027e-d821-4725-a264-c46249b8059b.png)
but neither led to the `// (undocumented)` being removed. This however did:
![image](https://user-images.githubusercontent.com/98850418/179266377-6373192a-715b-448f-bc2c-660ccb301ca8.png)
but I believe that is because of the text addition and not the `@see` tag getting recognized.

Also tried doing 
![image](https://user-images.githubusercontent.com/98850418/179268470-bb7404dd-700c-477f-81e8-cabe7cdf5ff8.png)
and 
![image](https://user-images.githubusercontent.com/98850418/179268539-8a72149a-edc3-40c5-9f68-27f22775d1b1.png)
and neither resulted in the removal of `// (undocumented)` tag, so my conclusion is it must be not recognizing the `@see` tag entirely.

Also the `@see` tag should be part of TSDoc grammar based on what I read [here](https://tsdoc.org/pages/tags/see/)

Reply:
Thanks. Sounds like it recognizes the tag but considers documentation consisting only of "@see" insufficient.
I guess we find+replace "@see" with "See" in these cases.

Reply:
Well, I guess not the newline, since that example would pass.... We'd just lose the formatting on the badge